tests/shared/controllers/feedback.test.js

```diff
@@ -43,9 +43,16 @@ test('POST /api/cb/feedback', tape => {
         .post('/api/cb/feedback')
         .set('authorization', token)
         .send(badPayload)
-        .expect(500)
-        .expect('Content-Type', /text/)
-        .end(async err => {
+        .expect(400)
```

Comment:
Looking a bit closer at this test, the reason the request fails is not because the payload is bad. Actually there's nothing wrong with the payload at all. It's that the email in the token is non-existent. 

This means that the request is well formed, but unauthorized. That's a `401`, and the router should check whether the email exists first and respond with an appropriate message.

Why aren't you using the authentication middleware on this route? It would do all this for you.

Reply:
Ahhh yeah, I should be using middleware 🙈 . Do you think I need an error test case, as this was checking the if the email grabbed from the jwt was linked to a valid cb. If it's running through middleware I guess this can never happen.